.claude/hooks/stop.py

```
@@ -0,0 +1,171 @@
+# dependencies = [
+#     "python-dotenv",
+# ]
+# ///
+
+import argparse
+import json
+import os
+import sys
+import subprocess
+from pathlib import Path
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)